cdc/sink/mq/producer/kafka/kafka.go

```diff
@@ -411,7 +412,10 @@ func AdjustConfig(
 	}
 
 	err = validateMinInsyncReplicas(admin, topics, topic, int(config.ReplicationFactor))
-	if err != nil {
+	// 'min.insync.replica' if invisible for Confluent Cloud. Moreover, replication-factor is
+	// required to be 3 for Confluent Cloud.
+	if err != nil && !strings.Contains(err.Error(),
```

Comment:
Should we hide that detail from entering validateMinInsyncReplicas instead of exposing it to the caller? Because it is a special case of this function.

Reply:
Good point. I will fix it later.